CombatTagPlus/src/main/java/net/minelink/ctplus/listener/TagListener.java

```diff
@@ -216,7 +216,8 @@ public void untagPlayer(PlayerKickEvent event) {
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void sendTagMessage(PlayerCombatTagEvent event) {
         // Do nothing if tag message is blank
-        String message = plugin.getSettings().getTagMessage();
+        String messageAttacker = plugin.getSettings().getAttackerTagMessage();
+        String messageVictim = plugin.getSettings().getVictimTagMessage();
```

Comment:
What are you trying to do here? You are defining two Strings and then never use them